custom_integrations/midea_ac_lan

```diff
@@ -0,0 +1 @@
+midea_lan
```

Comment:
We don't allow the use of symlinks for custom integrations (as described in the repository readme).

../Frenck

Reply:
So I'll duplicate data